test/e2e/definitions/limit-offset.ttl

```diff
@@ -0,0 +1,15 @@
+@base <http://example.org/pipeline/> .
+@prefix p: <https://pipeline.described.at/> .
+@prefix code: <https://code.described.at/> .
+@prefix base: <https://barnard59.zazuko.com/operations/base/> .
+
+<> a p:Pipeline, p:ReadableObjectMode ;
+  p:steps
+    [
+      p:stepList
+        (
+          [ base:streamValues ( "{ age: 34 }"^^code:EcmaScript "{ age: 23 }"^^code:EcmaScript "{ age: 10 }"^^code:EcmaScript) ]
```

Comment:
What do you think about the identifier `base:streamValues`? @giacomociti

Reply:
I like it. Even though the name of the function implementing it is completely different, I think it makes sense to use a higher-level language in the pipeline